pkg/network/http/model.go

```diff
@@ -62,9 +63,9 @@ func (tx *httpTX) StatusClass() int {
 	return (int(tx.response_status_code) / 100) * 100
 }
 
-// RequestLatency returns the latency of the request in microseconds
+// RequestLatency returns the latency of the request in seconds
 func (tx *httpTX) RequestLatency() float64 {
-	return float64((tx.response_last_seen - tx.request_started) / (1000))
+	return float64(uint64(tx.response_last_seen-tx.request_started) / uint64(time.Second))
```

Comment:
I don't think we want to do this integer division (otherwise you will truncate values < 1s